src/comet_llm/logging_messages.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# *******************************************************
+#   ____                     _               _
+#  / ___|___  _ __ ___   ___| |_   _ __ ___ | |
+# | |   / _ \| '_ ` _ \ / _ \ __| | '_ ` _ \| |
+# | |__| (_) | | | | | |  __/ |_ _| | | | | | |
+#  \____\___/|_| |_| |_|\___|\__(_)_| |_| |_|_|
+#
+#  Sign up for free at https://www.comet.com
+#  Copyright (C) 2015-2023 Comet ML INC
+#  This source code is licensed under the MIT license found in the
+#  LICENSE file in the root directory of this package.
+# *******************************************************
+
+from . import backend_error_codes
+
+SDK_ERROR_CODES_LOGGING_MESSAGE = {
```

Comment:
This dictionary shouldn't be in logging_messages.py, keep there only the messages themselves. This file should remain a simple storage of variables without any imports and knowledge about other modules.